eng/common/templates/job/job.yml

```diff
@@ -133,6 +133,7 @@ jobs:
       inputs:
         languages: ${{ coalesce(parameters.richCodeNavigationLanguage, 'csharp') }}
         environment: ${{ coalesce(parameters.richCodeNavigationEnvironment, 'production') }}
+        uploadRichNavArtifacts: ${{ coalesce(parameters.richCodeNavigationArtifactsUpload, 'true') }}
```

Comment:
I think it probably makes sense to make the default "false"

I also wonder if we could pass this is as an environment variable? I'm just thinking it'll be hard to keep making these input changes in arcade and waiting for them to propagate for each of our vars.. let me look up some docs and we can co-author a commit together?